### ISSUE TITLE
Location Intersection for targeted Alerting

### ISSUE THREAD

**Issue by waidyanatha** [sahana/SAMBRO#57]
There is a growing need to integrate the location intersection for targeted alerting. This is for Alert Subscribers who wish receive an alert for a specific location; like a SAR team that wants to ready if the alert applied to their area. 

Options 1) User indicates the Lx name (i.e. location name)
Option 2) User draws a polygon

In either case, we need to calculate whether the cap:alert:area:polygon on cap:alert:area:circle includes or excludes the particular subscriber. 

@flavour , @nursix I think @biplovbhandari is waiting on you to do this so he can integrate in SAMBRO? Do you see the need in other solutions. I am currently talking the the Finish Institute of Meteorology who is developing a river flood monitoring system for Nepal and they need SAMBRO to integrate through CAP to handle the dissemination part. Location intersection is one that they are asking for; what should I tell them?

**Comment by flavour** [comment]
This has been the only usecase for this need so far, but it certainly seems reusable.
We started working on it but then put it on hold as SAMBRO put it on hold.
Let's see where we are with the budget & then see if we can bring it back into focus

**Comment by waidyanatha** [comment]
Moved to CAPBroker missing features: http://eden.sahanafoundation.org/wiki/BluePrint/CAPBroker#Missingfeatures:
